.ci/jenkins_job_build.sh

```diff
@@ -52,22 +52,31 @@ mkdir -p $(dirname "${kata_repo_dir}")
 
 pushd "${kata_repo_dir}"
 
+
+# Variables needed when we test a PR.
 pr_number=
+target_branch=
+
+# Variable needed when a merge to a branch needs to be tested.
+branch=
 
 # $ghprbPullId and $ghprbTargetBranch are variables from
 # the Jenkins GithubPullRequestBuilder Plugin
-[ "${ghprbPullId}" ] && [ "${ghprbTargetBranch}" ] && pr_number="${ghprbPullId}"
+[ "${ghprbPullId}" ] && [ "${ghprbTargetBranch}" ] && export pr_number="${ghprbPullId}"
+
+
 
 if [ -n "$pr_number" ]
 then
+	export target_branch="${ghprbTargetBranch}"
```

Comment:
export target_branch="${ghprbTargetBranch:-master}" just in case is empty ? or fail if ghprbTargetBranch is not set.

Reply:
agree to make master default